scripts/visualize/portfolio_timeline.R

```diff
@@ -55,22 +54,7 @@ visualize.viz_y_sessions <- function(viz){
 
 }
 
-plot_timeline <- function(viz.data, range_text){
-
-  max_date <- Sys.Date() - 1
-  year_days = seq(max_date, length = 2, by = range_text)
-  year_days[1] <- year_days[1] - 1
-  full_dates <- seq(max_date, length = -as.numeric(diff(year_days))+1, by=-1)
-  empty_df <- data.frame(date = full_dates)
-
-  sub_data <- select(viz.data, date, sessions) %>%
-    group_by(date) %>%
-    summarize(sessions = sum(sessions, na.rm = TRUE)) %>%
-    data.frame() %>%
-    right_join(empty_df, by="date") %>%
-    arrange(desc(date))
-
-  sub_data$sessions[is.na(sub_data$sessions)] <- 0
```

Comment:
Just to defend the initial reason for leaving it in "visualize" step...the viz.yaml is >1000 lines long, and I've easily spent the greatest amount of time on this project trying to figure out what id goes where. By initially keeping the processes to essentially 3 (year/month/week), there was at least 1 thing that was easier to manage in that file. But...once we had the same set of code in 2 places, it became worth the effort of making new processes and keeping track of what viz step needs what processing.